docs/manuals/core/entities/pipeline-mgt.md

```diff
@@ -54,56 +54,26 @@ Taipy generates. A pipeline also holds various properties accessible as an attri
     current_month_data_node = pipeline.current_month
     ```
 
-# Get pipeline by id
+# Get a pipeline
 
-The method to access a pipeline is from its id by using the `taipy.get()^` method :
+There are two ways to retrieve a pipeline:
+
+The first method is by getting it from its parent scenario, using the pipeline name as an attribute.
```

Comment:
should we also mention by using the `pipelines` attribute?

Reply:
`scenario.pipelines` would return a dictionary right? It doesn't return the entity directly, but it can work. 

I'm not sure though.

Reply:
by that I meant `scenario.pipelines[pipeline_name]` :D